Description: relax package dependency versions and remove other os only deps
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-04-08
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,13 +30,4 @@
 default = []
 
 [target."cfg(not(any(target_os=\"windows\",target_os=\"android\",target_os=\"macos\")))".dependencies.x11-clipboard]
-version = "0.8.1"
-
-[target."cfg(target_os=\"android\")".dependencies.termux-clipboard]
-version = "0.1"
-
-[target."cfg(target_os=\"macos\")".dependencies.clipboard_macos]
-version = "0.1"
-
-[target."cfg(target_os=\"windows\")".dependencies.clipboard-win]
-version = "4.0.3"
+version = "0.9"
